src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,120 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final double COEFFICIENT = 1.5;
+    private static final int LIST_CAPACITY = 10;
+    private T[] data = (T[]) new Object[LIST_CAPACITY];
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
Not necessary initialize.
Its have default value 0.